File: python/Facade.py
from HotelApi import HotelApi
from AvionApi import AvionApi
class Facade():
    
    def __init__(self,fs,fv,o,d):
        self.hotel = HotelApi(fs, fv, o, d)
        self.avion=AvionApi(fs,fv,o,d)
        
    def buscar(self):
        self.avion.buscarVuelos()
        self.hotel.buscarHotel()
    
    
        
a=Facade("02/11/2020","03/01/2020","Uruguay","Caracas")
a.buscar()

File: Java/PatronFacade/src/PatronFacadeejemplo/Facade.java
package PatronFacadeejemplo;

public class Facade {
	private AvionApi avion;
	private HotelApi hotel;
	public Facade(String fs,String fv,String o,String d) {
		avion = new AvionApi(fs, fv, o, d);
		hotel = new HotelApi(fs,fv,o,d);
		
	}
	
	public void buscar() {
		avion.buscarvuelos();
		hotel.buscarHoteles();
	}

}
